components/analytics/true_up_review.tsx

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {useDispatch, useSelector} from 'react-redux';
+
+import moment from 'moment';
+
+import classNames from 'classnames';
+
+import {submitTrueUpReview, getTrueUpReviewStatus} from 'actions/hosted_customer';
+import {
+    isCurrentLicenseCloud,
+} from 'mattermost-redux/selectors/entities/cloud';
+
+import {
+    getSelfHostedErrors,
+    getTrueUpReviewProfile as trueUpReviewProfileSelector,
+    getTrueUpReviewStatus as trueUpReviewStatusSelector,
+} from 'mattermost-redux/selectors/entities/hosted_customer';
+
+import useCWSAvailabilityCheck from 'components/common/hooks/useCWSAvailabilityCheck';
+import CheckMarkSvg from 'components/widgets/icons/check_mark_icon';
+
+import './true_up_review.scss';
+import {GlobalState} from '@mattermost/types/store';
+import WarningIcon from 'components/widgets/icons/fa_warning_icon';
+import {isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
+import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
+import {getIsStarterLicense} from 'utils/license_utils';
+import store from 'stores/redux_store.jsx';
+
+const TrueUpReview: React.FC = () => {
+    const dispatch = useDispatch();
+    const getState = store.getState;
+    const isCloud = useSelector(isCurrentLicenseCloud);
+    const isAirGapped = !useCWSAvailabilityCheck();
+    const reviewProfile = useSelector(trueUpReviewProfileSelector);
+    const reviewStatus = useSelector(trueUpReviewStatusSelector);
+    const isSystemAdmin = useSelector(isCurrentUserSystemAdmin);
+    const license = useSelector(getLicense);
+    const isLicensed = license.IsLicensed === 'true';
+    const isStarter = getIsStarterLicense(license);
+    const licenseIsTrueUpEligible = isLicensed && !isCloud && !isStarter;
```

Comment:
For self-hosted we currently have starter GOV licenses so I believe starter would be a valid case. We are probably covered with the is cloud license check and we should be able to allow it to show for Starter. Is that correct @johndavidlugtu ?

Reply:
> For self-hosted we currently have starter GOV licenses so I believe starter would be a valid case.

In this question, is `Starter GOV` in the same group as `Free`?

If so, the ideal experience here is that only `Starter GOV` instances are able to do true-up reviews, as "true-up" as a concept is not relevant for Free instances.

> We are probably covered with the is cloud license check

Would need some further context here. For Cloud customers, there is no true-up as it is in self-hosted, as in Cloud, they are billed for additional users.

@marianunez @ConorMacpherson

Reply:
Right, the scenario for cloud users is covered here (no true-up will happen)

Gov licenses are definitely a different story. I think they would be covered under the current logic, but now that the starter check is removed, any starter license could attempt a true-up review.

If the GOV starter licenses should be eligible for true-up, then I'll add a new condition to check for starter GOV licenses specifically rather than just removing the check completely.

Reply:
> In this question, is `Starter GOV` in the same group as `Free`?

Feature-wise they are the same as free but contract-wise is different as they have premium support and not sure what else is involved. @johndavidlugtu we would need to talk to sales/csm to understand how true-up works with Gov Starter.

Reply:
Circling back on this, it's currently not practice to conduct true-up review for `Starter GOV` mainly due to bandwidth issues in the past. Will add more details here soon once I know more on whether we want that practice to be policy.

cc: @marianunez @ConorMacpherson

Reply:
Thanks @johndavidlugtu !